dbt_subprojects/hourly_spellbook/models/_sector/gas/fees/gas_fees.sql

```diff
@@ -22,6 +22,7 @@
                                     , "gnosis"
                                     , "ink"
                                     , "kaia"
+                                    , "katana"
```

Comment:
we need to add `katana` to the main list below too, where it lists the chains to be unioned into this final spell